pkgs/by-name/fr/freediameter/package.nix

```
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  pkg-config,
+  flex,
+  bison,
+  lksctp-tools,
+  libidn2,
+  libgcrypt,
+  gnutls,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "freediameter";
+  version = "1.5.0-unstable-2025-03-16";
+
+  src = fetchFromGitHub {
+    owner = "freeDiameter";
+    repo = "freeDiameter";
+    rev = "8e525acdfd439995f3e8e26d5a802fc4ad95d24c";
+    hash = "sha256-ai2R8scP++tdPh303RAl0qdIpehzFoyykAuAyl2w3MA=";
+  };
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    flex
+    bison
+    lksctp-tools
+    libidn2
+    libgcrypt
+    gnutls
+  ];
```

`strictDeps` ensures that build-time and run-time dependencies are separated:

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    bison
    cmake
    flex
    pkg-config
  ];

  buildInputs = [
    gnutls
    libgcrypt
    libidn2
    lksctp-tools
  ];
```